Assets/Scripts/Asteroid/AsteroidMovement.cs

```diff
@@ -26,16 +27,31 @@ private void OnCollisionEnter2D(Collision2D col)
     {
         if (col.gameObject.CompareTag("ShipBullet"))
         {
+            int shipHealth = GameObject.Find("Ship").GetComponent<Ship>().shipHealth;
             asteroidHealth -= 1;
             if (asteroidHealth == 0)
             {
                 Destroy(gameObject);
+
                 if (Random.Range(0.00f, 1.00f) < 0.25)
                 {
                     Instantiate(BulletsPowerUp, transform.position, transform.rotation);
+                    canAppear = false;
+                    StartCoroutine(NoAppear());
+                }
+                if (Random.Range(1.00f, 2.00f) < 1.10 && shipHealth < 10 && canAppear)
```

Comment:
Why are you using random range between 1 and 2? I think you can use 0 to 1 here as well.